tests/test_files.py

```diff
@@ -404,3 +425,31 @@ def test_PlateRecording__init__raises_error_if_given_a_file_with_version_v0_1():
                 )
             ]
         )
+
+
+def test_prof_get_raw_tissue_reading(generic_well_file_0_3_1):
```

Comment:
I missed this on the initial review, but this loop isn't really a great way to test performance when you are using caching, because the first iteration of the loop is going to be much longer than the subsequent ones.  So either you want a test that is focusing on how long it takes to do the initial retrieval, or you want a test focusing on how long it takes to do the retrieval after the first request